test/system/060-mount.bats

```diff
@@ -173,6 +173,35 @@ load helpers
     run_podman rm -t 0 -f $cid
 }
 
+@test "podman mount containers.conf" {
+    tmpfile1=$PODMAN_TMPDIR/volume-test1
+    random1=$(random_string 30)
+    echo $random1 > $tmpfile1
+
+    tmpfile2=$PODMAN_TMPDIR/volume-test2
+    random2=$(random_string 30)
+    echo $random2 > $tmpfile2
+
+    mountStr1=type=bind,src=$tmpfile1,destination=$tmpfile1,ro
```

Comment:
I find it a bit confusing that source and destination are the same.  Could we use another variable for the destination?
`mount_in_container="/tmp/$(random_string 10)"?

Reply:
Dest is now random